Calibration/HcalAlCaRecoProducers/python/alcaIsoTracksFilter_cfi.py

```diff
@@ -26,5 +25,9 @@
                                    ConeRadiusMIP     = cms.double(14.0),
                                    MinimumTrackP     = cms.double(20.0),
                                    MaximumEcalEnergy = cms.double(2.0),
-                                   IsolationEnergy   = cms.double(10.0),
+                                   MaxTrackP         = cms.double(8.0),
+                                   SlopeTrackP       = cms.double(0.05090504066),
```

Comment:
@bsunanda can you document from where does this magic number come from? Do you really need 10 digits?